src/pages/home/report/ReportActionCompose.js

```diff
@@ -90,6 +90,11 @@ class ReportActionCompose extends React.Component {
      * @param {String} newComment
      */
     updateComment(newComment) {
+        if (newComment.length <= 0) {
+            this.setState({textInputShouldClear: true});
+        } else {
+            this.setState({textInputShouldClear: false});
+        }
```

Comment:
I'm a little confused what clearing the text input has to do with anything.

Reply:
It doesn't seem like it would be necessary to clear the input when it has no value since it's already clear.